### ISSUE TITLE
Bot Stuck?

### ISSUE THREAD

**Issue by prendersnacks** [paco0x/amm-arbitrageur#11]
When running the bot it seems to only access the base token price. My only output has been as follows. Any idea why this might happen? 

Creating Typechain artifacts in directory typechain for target ethers-v5
Successfully generated Typechain artifacts!
2021-05-04 19:51:29 info: Load pairs from json
2021-05-04 19:51:30 info: Start arbitraging
2021-05-04 19:51:36 info: BNB price: $609.99
2021-05-04 20:51:30 info: BNB price: $627.05

Basically just been logging BNB price for twenty hours lol

**Comment by paco0x** [comment]
This may be caused by too many bots running so basically there's not too much price divergence.

**Comment by prendersnacks** [comment]
It just seems odd, I've messed with bot a bit and I've found that even removing things like my contract address it still works but does nothing but log BNB price every hour. I've edited variables to be more competitive and even to take a slight loss and it still seems to not output anything. Thanks for the quick response!

**Comment by paco0x** [comment]
If you are able to modify the bot code. You can set the minimum profit to a small value so there might be more opportunities for arbitrage. Note, the bot code is only for demonstration, it's not robust enough to be a production-ready product.
